Source/WebKit/Platform/cocoa/MediaCapability.mm

```diff
@@ -37,21 +38,11 @@
 
 using WebCore::RegistrableDomain;
 
-static RetainPtr<_SECapability> createPlatformCapability(const URL& url)
-{
-#if USE(EXTENSIONKIT)
-    if (_SECapability *capability = [get_SECapabilityClass() mediaWithWebsite:RegistrableDomain(url).string()])
-        return capability;
-#endif
-
-    UNUSED_PARAM(url);
-    return nil;
-}
-
 MediaCapability::MediaCapability(URL url)
     : m_url { WTFMove(url) }
-    , m_platformCapability { createPlatformCapability(m_url) }
 {
+    m_mediaEnvironment = adoptNS([[BEMediaEnvironment alloc] initWithWebPageURL:m_url]);
+    m_platformCapability = [BEProcessCapability mediaPlaybackAndCaptureWithEnvironment:m_mediaEnvironment.get()];
```

Comment:
ditto.

Reply:
I added this to the function body, since `m_platformCapability` is a member of the base class.

Thanks for reviewing!

Reply:
This data member should have been private in the base class. Then this constructor would either pass `platformCapability` to its base class constructor (ideally) or call a public setter on the base class (not as efficient but still OK).